packages/devextreme/js/__internal/core/m_devices.ts

```diff
@@ -4,6 +4,7 @@ import { EventsStrategy } from '@js/core/events_strategy';
 import $ from '@js/core/renderer';
 import type { Callback } from '@js/core/utils/callbacks';
 import Callbacks from '@js/core/utils/callbacks';
+import { when } from '@js/core/utils/deferred';
```

Comment:
I guess it was already migrated (other imports should be eventually fixed as well:
```suggestion
import { when } from '@ts/core/utils/m_deferred';
```

Reply:
`when` from `m_deferred` expects 0 arguments

Reply:
Looks like the declaration in the m_deferred.ts module should be updated, as nothing is changed in the export:

https://github.com/DevExpress/DevExtreme/blob/25_1/packages/devextreme/js/core/utils/deferred.js

```ts
export { fromPromise, setStrategy, Deferred, when } from '../../__internal/core/utils/m_deferred';
```Internal code should reference other internal code directly, without going through the public facade

General idea is: internal code should reference other internal code directly, without going through the public facade.